src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -14396,8 +14397,13 @@ void emitter::emitInsSveSanityCheck(instrDesc* id)
             break;
 
         case IF_SVE_BJ_2A: // ........xx...... ......nnnnnddddd -- SVE floating-point exponential accelerator
-        case IF_SVE_CH_2A: // ........xx...... ......nnnnnddddd -- SVE unpack vector elements
         case IF_SVE_HF_2A: // ........xx...... ......nnnnnddddd -- SVE floating-point reciprocal estimate (unpredicated)
+            assert(insOptsScalableAtLeastHalf(id->idInsOpt()));
```

Comment:
ah, so this is the one I missed. I assume the unit tests are working fine with this fix?

Reply:
Yes, fixes all the codegen tests.